src/main/java/TaskList.java

```diff
@@ -0,0 +1,51 @@
+public class TaskList {
+    private Task[] allTasks;
+    private int noOfTasks;
+
+    public TaskList(String task){
+        allTasks = new Task[100];
+        allTasks[0] = new Task(1, task);
+        noOfTasks = 1;
```

Comment:
Consider renaming the variable to something more intuitive such as numberOfTasks instead of abreviating number as no